packages/hardhat-chai-matchers/test/reverted/revertedWithPanic.ts

```diff
@@ -307,7 +307,7 @@ describe("INTEGRATION: Reverted with panic", function () {
           ).to.not.be.revertedWithPanic()
         ).to.be.eventually.rejectedWith(
           ProviderError,
-          "sender doesn't have enough funds to send tx"
```

Comment:
@fvictorio IIRC, we don't make any backwards compatibility guarantees about error messages, but to be sure I wanted to validate that this is alright to change?

Reply:
Yes, I'm fine with changing this